platform-ui-skin/src/main/webapp/skin/less/social/skin/social.less

```diff
@@ -548,3 +548,9 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     align-items: center;
     justify-content: space-around;
 }
+
+.mobile-chip-ellipsis{
```

Comment:
no need to add a new class here, please use existing CSS Class helpers (`text-truncate`) to apply ellipsis on elements, and then for each specific HTML element, add a dedicated CSS Class to add `max-width` property. In fact, sometimes you don't even need to add an exact `max-width` property value (like `130px` here), especially when an element is displayed in a single line and thus it mustn't overflow the current line (especially used for mobile). Consequently, using a generic `max-width: 100%` is sufficient to apply an ellipsis to avoid cutting texts.